odc/stats/plugins/l34_utils/l4_bare_gradation.py

```diff
@@ -0,0 +1,60 @@
+import xarray as xr
+from odc.stats._algebra import expr_eval
+from . import utils
+
+
+NODATA = 255
+
+
+def bare_gradation(xx: xr.Dataset, bare_threshold, veg_cover):
+
+    # Now add the bare gradation
+    fcp_nodaata = -999
```

Comment:
is it a typo? I don't see anywhere changing the fcp `nodata` value.

Reply:
I've removed the redundant statement.